packages/web-client/app/models/workflow.ts

```diff
@@ -15,6 +16,7 @@ export abstract class Workflow {
   name!: string;
   milestones: Milestone[] = [];
   epilogue: PostableCollection = new PostableCollection();
+  @tracked cancelation: PostableCollection | undefined;
```

Comment:
If `epilogue` is replaced, we will be showing `epilogue` both after a workflow is canceled or completed. This is nice because some of the behaviour from cancelling, we do want from completion too - eg. freezing a thread to prevent changes. I'm up for trying this out!